frontend/badges/legacyUser/query.sql

```diff
@@ -1,9 +1,9 @@
 SELECT
     DISTINCT `u`.`user_id`
 FROM
-	`Identity_Login_Log` AS `ill`
+	`Submissions` AS `s`
 INNER JOIN
-    `Identities` AS `i` ON `ill`.`identity_id` = `i`.`identity_id`
+    `Identities` AS `i` ON `s`.`identity_id` = `i`.`identity_id`
 INNER JOIN
     `Users` AS `u` ON `u`.`main_identity_id` = `i`.`identity_id`
-WHERE YEAR(`ill`.`time`) <= 2011;
\ No newline at end of file
+WHERE YEAR(`s`.`time`) <= 2011;
```

Comment:
puede esto mejor ser algo así como

```suggestion
WHERE `s`.`time` <= '2013-02-02 13:12:24';
```

? eso es bastante más barato porque no hay que hacer interpretación del tiempo ni aritmética de fechas, y además resulta que el primer envío se hizo en `2012-02-02 13:12:24`.

Reply:
Ahora lo modifico!